code/game/mecha/equipment/tools/medical_tools.dm

```diff
@@ -488,7 +488,7 @@
 	return 1
 
 /obj/item/mecha_parts/mecha_equipment/medical/syringe_gun/proc/add_known_reagent(r_id,r_name)
-	if(!(r_id in known_reagents))
+	if(!(r_id in known_reagents) && (r_id in standard_medicines))

```

Comment:
I intended to, but unfortunately enough not all admin chemicals (such as xenomicrobes) are actually in that list.